src/website/app/demos/routes.js

```diff
@@ -17,58 +17,86 @@
 /* @flow */
 export default {
   button: {
+    description:
+      'A Button should be used whenever you need to trigger an action in your app. The color of the button should be chosen according to the intent of the action.',
     slug: 'button',
     title: 'Button'
   },
   card: {
+    description:
+      'Cards are used to group different kinds of elements with actions attached. Use a Card to contain content that might not fit neatly into a table row or grid cell.',
     slug: 'card',
     title: 'Card'
   },
   'card-block': {
+    description:
+      'CardBlock is used to help lay out content that’s not a title or an image in the body of the Card.',
     slug: 'card-block',
     title: 'CardBlock'
   },
   'card-image': {
+    description:
+      'Use a CardImage to reinforce the intent of the Card. Images shouldn’t be used alone in a Card, but should be paired with a call to action and/or a CardTitle.',
     slug: 'card-image',
     title: 'CardImage'
   },
   'card-title': {
+    description:
+      'Use a CardTitle when you need a consistently styled heading for your Card. Use a subtitle to provide supporting information for the data displayed in the Card.',
     slug: 'card-title',
     title: 'CardTitle'
   },
   dropdown: {
+    description:
+      'Use Dropdown for extra functionality, not for primary actions, since the options are hidden from the user until interaction.',
     slug: 'dropdown',
     title: 'Dropdown'
   },
   icon: {
+    description:
+      'Icons can symbolize actions and objects in your interace. Use icons in combination with labels to help users more quickly process your UI.',
```

Comment:
interface is misspelled :/

Reply:
hah, that means it's spelled wrong in the usage section too. I'll add a fix there.